src/components/SpendingItem/SpendingItem.jsx

```diff
@@ -49,13 +49,15 @@ const SpendingItem = ({ category, note, labels, createdAt, amount, currency }) =
         return null;
     }
   };
-
   const getFormatDate = (date) => {
     const formatDate = new Date(date);
     return `${
       shortMonthNames[formatDate.getMonth()]
     } ${formatDate.getDate()}, ${formatDate.getFullYear()}`;
   };
+  const deleteSpending = (spendingID) => {
```

Comment:
In most cases handler for `on...` events can be called `handle...`. E.g. `handleDeleteSpending`